Templates/MonoGame.Templates.VSExtension/MonoGame.Templates.VSExtension.csproj

```diff
@@ -58,16 +26,21 @@
     <Reference Include="System" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.10.*" ExcludeAssets="runtime" />
-    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.10.*" />
+    <PackageReference Include="MessagePack" Version="2.5.187" />
+    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.11.40262" ExcludeAssets="runtime" />
```

Comment:
Which versions of the IDE have you tested? I don't have access to test, but we should work on the current VS2022 and I guess VS2019?

Reply:
I don't have VS2019, but do have all the VS2022s.

Reply:
@dellis1972 The extensions install in all the VS2022 IDEs, but do not build out of the box. I get the following error for all the projects I've created, namely, Android, Desktop, WinDX. Any suggestions?

<img width="674" alt="image" src="https://github.com/user-attachments/assets/32aec06c-124b-440c-a02e-732396f4e46f">

Reply:
`dotnet` is probably not part of the PATH env variable and that is causing the issue.

On macOS / Linux this is default and automated.

On Windows I believe there is a checkbox that you need to check for it to be so.

Reply:
@harry-cpp I'm not sure that is the issue, because When I load another sln, like the Start-Kit template sln, things just work.